src/submit-modal/use-aggregate-data-exchange-mutation.js

```diff
@@ -25,15 +26,15 @@ const submitReducer = (state, action) => {
                 error: null,
                 loading: false,
                 called: true,
-                dataSubmitted: true,
+                dataSubmitted: moment().format(),
```

Comment:
dataSubmitted is changed from a boolean (has data been submitted) to a string (when the data was submitted, for use in the clipboard text)